### ISSUE TITLE
PreTrain Model

### ISSUE THREAD

**Issue by Orbis36** [Yiming992/Test-Linking-Segments-Text-Localization#2]
Hello!
I am using the method you listed in Route II
I found that the VGG16 pretrained on SynthText does not include the layers after like ‘conv8_2’ to extract the feature and the parameters in convolutional predictors.
Does it mean that I can acquire the same loss(around 7) at last just with fine-tune?
Thanks for any help
